eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/consumer/EventMeshHttpConsumer.java

```diff
@@ -99,7 +99,7 @@ public void subscribe(final List<SubscriptionItem> topicList, final String subsc
             final String subRes = HttpUtils.post(httpClient, target, subscribeParam);
             final EventMeshRetObj ret = JsonUtils.parseObject(subRes, EventMeshRetObj.class);
             if (ret.getRetCode() != EventMeshRetCode.SUCCESS.getRetCode()) {
-                throw new EventMeshException(ret.getRetCode(), ret.getRetMsg());
```

Comment:
Sorry, the number of issue description lines may be incorrect due to the code update. The places to be changed here are as follows:
![image](https://user-images.githubusercontent.com/91315508/221463072-646adffd-5650-455e-a397-374306107446.png)
![image](https://user-images.githubusercontent.com/91315508/221463103-6f455e22-6f3b-407b-82bd-25781f7feea4.png)
![image](https://user-images.githubusercontent.com/91315508/221463147-634ce6a3-6549-443b-b201-54c3dd6920b8.png)